pages/styles/global.css

```diff
@@ -293,6 +297,31 @@ header a p {
 	transform: translateY(-50%)
 }
 
+main a, 
+footer a {
+    text-decoration: underline;
+    color: var(--link-color);
+}
+
+.main__report {
```

Comment:
Thank you for transferring the "Report a problem..." button CSS to the global stylesheet. Though the button I believe it should be an `<a>` (anchor) tag to accommodate users who don't have JavaScript enabled. 

The work for the button should be in a separate branch for whoever takes it.